### PR TITLE
Improvements to CW canned messages

### PR DESCRIPTION
**Pull request by Tali64**
I've rewritten many of the canned CW request handling messages to be more consistent as well as added a new one for duplicates of requests that have already been approved.

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -817,20 +817,21 @@
...
 				'Okay request' => 'Okay-ish. Description doesn\'t meet our requirements, but in this case the sitename, URL, and categorisation suggest this is a wiki that would follow the Content Policy made clear by the preceding fields, and it is conditionally approved as such. Please be advised that if your wiki deviates too much from this approval, remedial action can be taken by a Steward which includes wiki closure and potential revocation of wiki requesting privileges, if necessary. Please ensure your wiki complies with all aspects of Content Policy and Code of Conduct at all times. Thank you.',
-				'Categorised as private' => 'The purpose and scope of your wiki is clear enough. Please ensure your wiki complies with all aspects of the Content Policy and Code of Conduct at all times or it may be closed. Please also note that I have categorised your wiki as "Private". Thank you.',
+				'Categorised as private' => 'The purpose and scope of your wiki is clear enough. Please ensure your wiki complies with all aspects of the Content Policy and Code of Conduct at all times or it may be closed. Please also note that I have categorized your wiki as "Private". Thank you.',
```
Comment (RhinosF1) [inline LocalSettings.php:820]: Please use one spelling throughout

**Review by RhinosF1 — CHANGES_REQUESTED** [review]

File: LocalSettings.php [hunk 2]
```diff
@@ -874,21 +874,22 @@
...
-				'Categorised as private' => 'The purpose and scope of your wiki is clear enough. Please ensure your wiki complies with all aspects of the Content Policy and Code of Conduct at all times or it may be closed. Please also note that I have categorised your wiki as "Private". Thank you.',
+				'Okay request' => 'Okay-ish. Description doesn\'t meet our requirements, but in this case the sitename, URL, and categorization suggest this is a wiki that would follow the Content Policy made clear by the preceding fields, and it is conditionally approved as such. Please be advised that if your wiki deviates too much from this approval, remedial action can be taken by a Steward which includes wiki closure and potential revocation of wiki requesting privileges, if necessary. Please ensure your wiki complies with all aspects of Content Policy and Code of Conduct at all times. Thank you.',
```
Comment (R4356th) [inline LocalSettings.php:877]: Why are you changing this?

File: LocalSettings.php [hunk 3]
```diff
@@ -874,21 +874,22 @@
...
-				'Invalid sitename/subdomain (obscene wording)' => 'The scope and purpose of the wiki seem clear enough. However, the requested wiki name or subdomain is in violation of our Content Policy which prohibits obscene wording in wiki names and subdomains. Please change it to something that is better. Thank you.',
+				'Needs more details' => 'Can you give us more details on the purpose for, scope of, and topic of your wiki, ideally in at least 2-3 sentences? Please go back into your original request and add to, but do not replace, your existing description. Thank you.',
+				'Invalid or unclear subdomain' => 'The scope and purpose of your wiki seem clear enough; however, the requested subdomain is either invalid, too generic, conveys a Miraheze affiliation, or suggests that the wiki is an English language or multilingual wiki when it is not. Please change it to something that better reflects your wiki\'s purpose and scope. Thank you.',
```
Comment (R4356th) [inline LocalSettings.php:882]: This change just makes the sentence bigger.

File: LocalSettings.php [hunk 4]
```diff
@@ -874,21 +874,22 @@
...
+				'Invalid sitename/subdomain (obsence wording)' => 'The scope and purpose of your wiki seem clear enough; however, the requested wiki name or subdomain is in violation of our Content Policy, which prohibits obsence wording in wiki names and subdomains. Please change it to something that is appropriate. Thank you.',
```
Comment (R4356th) [inline LocalSettings.php:883]: "obsence" is not a word (I had to do #5326 to fix this previously), and you are once again creating a big sentence. The "better" -> "appropriate" change makes sense to me though.

File: LocalSettings.php [hunk 5]
```diff
@@ -903,11 +904,11 @@
...
 				'Content Policy (violence, hatred or harrassment)' => 'Declining per Content Policy provision, "Miraheze does not host wikis that promote violence, hatred, or harassment against a person or group of people." Thank you for your understanding.',
 				'Content Policy (Wikimedia-like wikis/forks)' => 'Declining per Content Policy provision, "Direct forks and forks where a substantial amount of content is copied from a Wikimedia project are not allowed." Thank you for your understanding.',
-				'Reception wiki' => 'Declining per resolution of a Request for Comment, "No new reception wikis will be accepted on the platform." Thank you for your understanding.',
-				'Author request' => 'Declined at the request of the wiki requester.',
+				'Reception wiki' => 'Declining per resolution of a Request for Comment, "No new reception wikis will be accepted on the platform indefinitely." Thank you for your understanding.',
+				'Author request' => 'Declined at the request of the wiki requestor.',
```
Comment (R4356th) [inline LocalSettings.php:908]: Why do you want to use a rarely used spelling instead of a perfectly valid one?

**Review by R4356th — CHANGES_REQUESTED** [review]
Please fix these issues.

File: LocalSettings.php (hunk 4 above)
Comment (Tali64) [reply LocalSettings.php]: Fixed; not sure how "obsence" got in there.

File: LocalSettings.php [hunk 6]
```diff
@@ -874,21 +874,22 @@
...
+				'Needs more details' => 'Can you give us more details on the purpose for, scope of, and topic of your wiki, ideally in at least 2-3 sentences? Please go back into your original request and add to, but do not replace, your existing description. Thank you.',
+				'Invalid or unclear subdomain' => 'The scope and purpose of your wiki seem clear enough; however, the requested subdomain is either invalid, too generic, conveys a Miraheze affiliation, or suggests that the wiki is an English language or multilingual wiki when it is not. Please change it to something that better reflects your wiki\'s purpose and scope. Thank you.',
+				'Invalid sitename/subdomain (obscene wording)' => 'The scope and purpose of your wiki seem clear enough; however, the requested wiki name or subdomain is in violation of our Content Policy, which prohibits obsence wording in wiki names and subdomains. Please change it to something that is appropriate. Thank you.',
```
Comment (R4356th) [inline LocalSettings.php:883]: You missed this use of "obsence".

Comment (Tali64) [reply LocalSettings.php]: Fixed.

File: LocalSettings.php (hunk 2 above)
Comment (Reception123) [reply LocalSettings.php:877]: I don't think there's a particular need to change to US spelling, even if MH will now be based in the US. Citing Wikipedia policies can be controversial but https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style#National_varieties_of_English is probably helpful here.

**Comment by Reception123** [comment]
Generally it's a bit unclear to me why some of these changes would be "improvements" or better than the original versions. Explanations for why the new version should be preferred would be useful.

**Comment by Tali64** [comment]
I made a lot of the changes based on my personal preference of how they should be worded, which I now realize may not match up with the best way to word them; I've reverted some of my changes that added little to no benefit to comprehensibility and conciseness.

**Comment by Tali64** [comment]
In my latest commit, I added "mywiki.miraheze.org" to the list of disallowed subdomains and added a commented-out canned response for the Content Policy's additional restrictions that may soon be in effect; if anyone has any suggestions, please comment.

File: LocalSettings.php [hunk 7]
```diff
@@ -905,6 +906,7 @@
 				'Content Policy (violence, hatred or harrassment)' => 'Declining per Content Policy provision, "Miraheze does not host wikis that promote violence, hatred, or harassment against a person or group of people." Thank you for your understanding.',
 				'Content Policy (Wikimedia-like wikis/forks)' => 'Declining per Content Policy provision, "Direct forks and forks where a substantial amount of content is copied from a Wikimedia project are not allowed." Thank you for your understanding.',
 				'Reception wiki' => 'Declining per resolution of a Request for Comment, "No new reception wikis will be accepted on the platform indefinitely." Thank you for your understanding.',
+				/* 'Content Policy (additional restrictions)' => 'Declining per the Content Policy's additional restrictions, which includes the topic of your wiki. Thank you for understanding.', */
```
Comment (R4356th) [inline LocalSettings.php:909]: Nitpick: All other responses say "your understanding".

**Comment by Universal-Omega** [comment]
This has conflicts

File: LocalSettings.php [hunk 8]
```diff
@@ -852,49 +852,52 @@
...
 				'Content Policy (forks)' => 'Declining per Content Policy provision, "Direct forks of other Miraheze wikis where no attempts at mediations are made are not allowed." Thank you for your understanding.',
-				'Content Policy (illegal UK activity)' => 'Declining per Content Policy provision, "Miraheze does not host any content that is illegal in the United Kingdom." Thank you for understanding. If you believe this decline reason was used incorrectly, please address this with the declining wiki creator on their user talk page first before escalating your concern to the Stewards\' noticeboard. Thank you.',
+				'Content Policy (illegal UK activity)' => 'Declining per Content Policy provision, "Miraheze does not host any content that is illegal in the United Kingdom." Thank you for understanding. If you believe this decline reason was used incorrectly, please address this with the declining wiki creator on their user talk page first before escalating your concern to Steward requests. Thank you.',
```
Comment (Universal-Omega) [inline LocalSettings.php:886]: Should be US

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 8 above)
Comment (Tali64) [reply LocalSettings.php]: Done.

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**